src/components/tx/TxData.js

```diff
@@ -463,10 +463,13 @@ class TxData extends React.Component {
 
     const renderGraph = (graphIndex) => {
       return (
-        <div className="d-flex flex-column flex-lg-row align-items-start mb-3 common-div bordered-wrapper w-100">
+        <div key={graphIndex} className="d-flex flex-column flex-lg-row align-items-start mb-3 common-div bordered-wrapper w-100">
           <div className="mt-3 graph-div" key={`graph-${this.state.graphs[graphIndex].name}-${this.props.transaction.hash}`}>
             <label className="graph-label">{this.state.graphs[graphIndex].label}:</label>
-            <a href="true" className="ml-1" onClick={(e) => this.toggleGraph(e, graphIndex)}>{this.state.graphs[graphIndex].showNeighbors ? 'Click to hide' : 'Click to show'}</a>
+            { this.props.transaction.parents && this.props.transaction.parents.length
+            ? <a href="true" className="ml-1" onClick={(e) => this.toggleGraph(e, graphIndex)}>{this.state.graphs[graphIndex].showNeighbors ? 'Click to hide' : 'Click to show'}</a>
+            : null
+            }
```

Comment:
When trying to show a transaction that has not been mined it was showing a bright red error page inside the graph area.
To minimize this, we hide the button that requests the graph to be rendered, indicating that it has no data to show.